### PR TITLE
fix: extrinsic for exchange state

### LINKED ISSUE #546: Functionality needed for Exchange state
- [x] Extrinsic to revert ExchangeState
- [x] Make use of ExchangeState except in claim_withdraw and claim_fee

### PR DESCRIPTION
**Pull request by felixfaisal**
This PR resolves #546

### CODE REVIEW

**Review by Gauthamastro — DISMISSED** [review]

**Comment by serhii-temchenko** [comment]
Please add tests for each added extrinsic which tests `ExchangeNotOperational` error.

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]
Please add tests for each added extrinsic which tests `ExchangeNotOperational` error.

File: pallets/ocex/src/lib.rs [hunk 1]
```diff
@@ -756,6 +769,14 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Extrinsic to update ExchangeState
+		#[pallet::weight(1000000)]
+		pub fn set_exchange_state(origin: OriginFor<T>, state: bool) -> DispatchResult {
+			T::GovernanceOrigin::ensure_origin(origin)?;
+			<ExchangeState<T>>::put(state);
```
Comment (felixfaisal) [inline pallets/ocex/src/lib.rs:830]: Emit event for setting exchange state

Comment (felixfaisal) [reply pallets/ocex/src/lib.rs:830]: Updated ✅

**Review by serhii-temchenko — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**